src/compositions/fission.ts

```diff
@@ -0,0 +1,52 @@
+import { Components } from "../components.js"
+import { Configuration } from "../configuration.js"
+import * as Config from "../configuration.js"
+
+import * as Account from "../components/account/fission.js"
+import * as Agent from "../components/agent/web-crypto-api.js"
+import * as Channel from "../components/channel/fission.js"
+import * as Depot from "../components/depot/fission.js"
+import * as DNS from "../components/dns/dns-over-https.js"
+import * as Identifier from "../components/identifier/web-crypto-api.js"
+import * as Manners from "../components/manners/default.js"
+import * as Storage from "../components/storage/indexed-db.js"
+
+/**
+ * The default Fission stack using web crypto auth and IPFS.
+ */
+export async function components(
+  settings: Configuration & {
+    environment?: string
```

Comment:
`environment` is unused. Should this be passed to the account implementation?

Reply:
Ah yup, good catch! Fixed it 👍